inference-engine/ie_bridges/java/cpp/openvino_java.hpp

```diff
@@ -0,0 +1,123 @@
+#include <jni.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+//
+// IECore
+//
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_IECore_ReadNetwork1(JNIEnv *, jobject, jlong, jstring, jstring);
```

Comment:
why do we use OpenVINO while it wraps only IE?

Reply:
Do you mean, that import for java bindings should look like ```import org.intel.openvino.inference_engine.*;```?